### PR TITLE
Create the from to component

### PR DESCRIPTION
**Pull request by diegoAndres01**
Create the from to component 
Card Jira 
https://inube.atlassian.net/browse/CB-120

### CODE REVIEW

File: src/components/feedback/InputRange/index.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    }
+
+    const handleChangeTo = (e: React.ChangeEvent<HTMLInputElement>) => {
+        setInputValueTo(e.target.value);
+        handleInputChangeTo(e);
+    }
+    
+
+    return (
+        <>
```
Comment (Andresbl123) [inline src/components/feedback/InputRange/index.tsx:63]: creo que aquí podemos usar un contenedor, esto permitiría poder ajustar temas de layout y responsive, ya que ahora están dejando de algunos espacios que lo rompen 
image

**Review by Andresbl123 — APPROVED** [review]
Cumple requerimiento

File: src/components/feedback/InputRange/index.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
+import { useState } from "react";
+import { Toggle } from "@inubekit/toggle";
+import { Textfield } from "@inubekit/textfield";
+import { Grid } from "@inubekit/grid";
+import { Datefield } from "@inubekit/datefield";
+import { Stack } from "@inubekit/stack";
+
+
+export interface InputRangeProps {
+    checked?: boolean;
```
Comment (carin91) [inline src/components/feedback/InputRange/index.tsx:10]: Relocate the optional props and the bottom

File: src/components/feedback/InputRange/index.tsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    handleToggleChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+    handleInputChangeFrom: (event: React.ChangeEvent<HTMLInputElement>) => void;
+    handleInputChangeTo: (event: React.ChangeEvent<HTMLInputElement>) => void;
+    id: string;
+    label: string;
+    valueToggle?: string;
+    valueFrom?: string | number;
+    valueTo?: string | number;
+    typeInput: ITextfieldInputType;
```
Comment (carin91) [inline src/components/feedback/InputRange/index.tsx:19]: Relocate the mandatory props at the top

File: src/components/feedback/InputRange/index.tsx [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+    const handleChangeTo = (e: React.ChangeEvent<HTMLInputElement>) => {
+        setInputValueTo(e.target.value);
+        handleInputChangeTo(e);
+    }
+    
+
+    return (
+        <>
+            <Toggle
```
Comment (carin91) [inline src/components/feedback/InputRange/index.tsx:64]: i would create a new Toggle component to wrap the others

File: src/components/feedback/InputRange/index.tsx [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+                                onChange={handleChangeFrom}
+                                required={required}
+                                size="compact"
+                                type="number"
+                                value={inputValueFrom}
+                            />
+                        )}
+                        {
+                            typeInput === "date" && (
+                                <Datefield
```
Comment (carin91) [inline src/components/feedback/InputRange/index.tsx:90]: Despite the wireframes, the "from" "to" component is not for dates.
Could be money or a general number.
As an example "Antiguedad de 5 años a 10 años"
"Desembolso de $10'000.000 a $20'000.000"
i would include a prop to evaluate whether is money or not.

File: src/components/feedback/InputRange/index.tsx [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+                size="large"
+                value={valueToggle}
+            />
+            {toogleCheck && (
+                <Grid templateColumns="repeat(2, 1fr)" margin="10px 0px" gap="12px">
+                    <Stack>
+                        {typeInput === "number" && (
+                            <Textfield
+                                id={`${id}TextFieldFrom`}
+                                label="Desde"
```
Comment (carin91) [inline src/components/feedback/InputRange/index.tsx:80]: The label should be rendered from a config file, not a magical word.

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/components/feedback/InputRange/stories/InputRange.stories.tsx [hunk 7]
```diff
@@ -0,0 +1,53 @@
+import { Meta, StoryObj } from "@storybook/react";
+
+import { parameters, props } from "./props";
+import {
+  InputRange, InputRangeProps
+} from "..";
+
+
+const meta: Meta<typeof InputRange> = {
+  title: "components/InputRange",
```
Comment (carin91) [inline src/components/feedback/InputRange/stories/InputRange.stories.tsx:10]: Relocate the story to "components/inputs/inputRange"
image

**Review by carin91 — APPROVED** [review]

**Pull request merged**